toolkit/pythonit_toolkit/api/client.py

```diff
@@ -0,0 +1,54 @@
+import os
+from typing import Any, Dict, Optional
+
+import httpx
+import jwt
+from pythonit_toolkit.headers import SERVICE_JWT_HEADER
+
+INTERNAL_JWT_SECRET = os.getenv("INTERNAL_JWT_SECRET", "a-very-secret-secret")
+
+
+class Client:
+    url = "/internal-api"
+
+    async def execute(
+        self,
+        query: str,
+        variables: Optional[Dict[str, Any]] = None,
+    ):
+        token = jwt.encode({"some": "payload"}, INTERNAL_JWT_SECRET, algorithm="HS256")
```

Comment:
why `some: payload`?

Reply:
also this token isn't generated correctly. Ideally it should contain for which service it was generated and what is the destination.

Also each service has it is own internal api, so `url` is not unique

Reply:
This is how it is generated on the Gateway: https://github.com/pythonitalia/pycon/blob/8900d9a24ad9b058dee638f206c5dedae183c36d/gateway/pastaporto/user-info.ts#L20

They are also very short-lived, 1 minute max

Instead of having the token generation in the execute method, I would create a method in the `tokens.py` file (if we have it) or at least some different file